benchmark_test/scripts/benchmark-test-setup.sh

```diff
@@ -43,6 +42,65 @@ install_local_path_provisioner(){
 }
 
 
+generate_longhorn_yaml_manifest(){
```

Comment:
This is actually similar to test_framework/scripts. For now, it's OK, but probably you can think about making them shareable in the future, so that will be easy for you for later refactoring.